scipy/special/_extra_special.h

```diff
@@ -3,8 +3,12 @@
 #include <complex>
 #include <numpy/npy_math.h>
 
+#include "_binom.h"
 #include "_lambertw.h"
 
+inline double binom(double n, double k) {
```

Comment:
Do we want this to be `npy_double` or just `double`?

Reply:
I don't think it matters for `double`. It seems like the convention in other parts of the code base is to just have `double`. Check out [specfun_wrappers](https://github.com/scipy/scipy/blob/main/scipy/special/specfun_wrappers.h) for instance.